test/core_test/api_test.rb

```diff
@@ -56,6 +56,7 @@ def wait_until(iterations = 10, interval = 0.2)
 
         post "/tasks/#{triggered.id}/cancel"
         triggered.finished.wait(5)
+        wait_until { WORLD.persistence.load_execution_plan(triggered.id).state == :stopped }
```

Comment:
Is this a typo or do we really have a `wait_until` function in the tests?

Reply:
yes, we have it, see 5 lines above

Reply:
Oh, I see, completely forgot about it